src/resources/js/tickets.js

```diff
@@ -635,6 +636,14 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 			return;
 		}
 
+		// setting triggerHandler as a variable is needed to return a new value of attendeeLablesIsValid if needed.
+		attendeeLablesIsValid = $tribe_tickets.triggerHandler( 'attendee-label-validation.tribe', [ attendeeLablesIsValid ] );
```

Comment:
Can we possibly leave triggerHandler and the variable as something more generic so that we can also use it for other types of validation we may need?

This is referring to the ticketForm, so maybe using that could be a good idea. Something like `ticketFormIsValid` or similar. The trigger also needs to be camelCased.

Then on ET+ we can use the attendeeLabel thing, as there we're validating the labels. But here it'd make more sense to have something generic that we can hook on to make other types of validations.

Reply:
I updated this. But I just used an `additionalValidation` term because there is already validation happening above what I'm doing on the form. Felt this was at least something that didn't conflict with the first validation but had a more generic name